src/libstd/Cargo.toml

```diff
@@ -47,7 +47,7 @@ hermit-abi = { version = "0.1.10", features = ['rustc-dep-of-std'] }
 wasi = { version = "0.9.0", features = ['rustc-dep-of-std'], default-features = false }
 
 [features]
-default = ["std_detect_file_io", "std_detect_dlsym_getauxval"]
+default = ["std_detect_file_io", "std_detect_dlsym_getauxval", "panic-unwind"]
```

Comment:
This is somewhat surprising to me, what's the explanation?

Reply:
We need to link to some panic runtime for std to build and I believe bootstrap always builds std with panic-unwind linked in today.

Reply:
Wouldn't this negatively affect some `-Zbuild-std` uses?